endpoint.h

```diff
@@ -22,6 +22,9 @@
 #include "comm.h"
 #include "pollable.h"
 
+#define READ_MESSAGE_OK 1
+#define READ_MESSAGE_UNIDENTIFIED 2
```

Comment:
I can do that, but isn't it weird to compare it with some old constant, like `-EINVAL` or `0`, and if not lesser than zero, with some enum?